src/js/index.js

```diff
@@ -456,7 +456,7 @@ function init(config) {
 
 // GETSTATE FUNCTION //
 
-function getState() {
+export function getState() {
```

Comment:
@liamaharon What is the reason for this change?

Reply:
As mentioned in Slack this is redundant, my mistake for leaving it in. Good pickup.